addons/popochiu/engine/objects/gui/components/dialog_text/dialog_text.gd

```diff
@@ -169,46 +173,37 @@ func _modify_size(_msg: String, _target_position: Vector2) -> void:
 
 
 ## Creates a RichTextLabel to calculate the resulting size of this node once the whole text is shown.
+## Uses a RichTextLabel to provide the "get_parsed_text" function, and a label within it to work out
+## the minimum size. Calculating the size from just the RichTextLabel does not work.
```

Comment:
If you want to replace the description of the function, please remove the previous one. Also, I suggest a small change to make it clear what the function really does.

```suggestion
## Uses a [RichTextLabel] and a [Label] to work out the minimum size since calculating it from just
## the [RichTextLabel] does not work.
```